src/main/webapp/script.js

```diff
@@ -69,11 +77,49 @@ function getPlaceDetails(map, restaurantChoice){
         });
         choiceMarkers.push(marker);
 
-        // If restaurant choice is null
-        if (Object.is(place,null)) {
-            document.getElementById("places").innerHTML = "Oops you're too picky, choose another location or change your filters!";
+        console.log("place " + place.name);
+        if (place.name) {
+            document.getElementById("result-restaurant").style.visibility = 'visible';
+    
+            document.getElementById("places").innerHTML = place.name;
+            if(place.formatted_phone_number) {
+                document.getElementById("address").innerHTML = '<i class="lni lni-restaurant"></i> Address : ' + place.formatted_address;
+            } else {
+                document.getElementById("address").innerHTML = "";
+            }
+            if(place.formatted_phone_number) {
+                document.getElementById("phone").innerHTML = '<i class="lni lni-phone"></i> Phone : ' + place.formatted_phone_number;
+            } else {
+                document.getElementById("phone").innerHTML = "";
+            }
+            if (place.website) {
+                if (place.website.length > 80) {
+                    document.getElementById("website").innerHTML = 'Website : ' + place.website.substring(0, 80);
+                } else {
+                    document.getElementById("website").innerHTML = 'Website : ' + place.website;
+                }
+            } else {
+                document.getElementById("website").innerHTML = "";
+            }
+            if (place.rating) {
+                // <i class="lni lni-star"></i>
+                document.getElementById("rating").innerHTML = '<i class="lni lni-star"></i> Rating : ' + place.rating;
+            } else {
+                document.getElementById("rating").innerHTML = "";
+            }
+            // if (place.icon) {
+            //     document.getElementById("icon").src = place.icon;
+            // } else {
+            //     document.getElementById("icon").src = "/images/food.png";
+            // }
+            
         } else {
-            document.getElementById("places").innerHTML = place.name + "\n\r" + place.formatted_address;
+            console.log("No Restaurant found")
+            document.getElementById("places").innerHTML = "Oops you're too picky, please try another search (choose another location, widen the radius, or change your filters)!";
+            document.getElementById("address").innerHTML = "";
+            document.getElementById("phone").innerHTML = "";
+            document.getElementById("website").innerHTML = "";
+            document.getElementById("rating").innerHTML = "";
         }
```

Comment:
when no results are found, the map recenters on the ocean

![Screen Shot 2020-08-30 at 7 52 26 PM](https://user-images.githubusercontent.com/31350668/91658499-3da2ed00-eafb-11ea-83ea-c22d9adcef47.png)

Reply:
`Uncaught TypeError: Cannot read property 'place_id' of undefined
    at getPlaceDetails (script.js:60)
    at createMarkers (script.js:274)
    at script.js:220
    at m$._.t.Xh (places_impl.js:71)
    at places_impl.js:73
    at places_impl.js:46
    at kq.e [as o] (places_impl.js:29)
    at Object.c [as _32960d] (common.js:67)
    at PlaceService.FindPlaces?1m6&1m2&1d24.763685121507294&2d120.97046516432567&2m2&1d24.781651427189686&2d120.99025233567428&2sen-US&6srestaurant&19u4&20u4&29e3&36m1&2e1&1032e0&callback=_xdc_._32960d&key=AIzaSyBnHjJrXSDt16Vgg76YOzIBK1KTid0cfeI&token=18634:1`